packages/forms-web-app/src/pages/projects/section-51/index/_utils/advice-helpers.js

```diff
@@ -16,7 +16,8 @@ const getAdviceName = ({ organisation, firstName, lastName }, i18n) => {
 	);
 };
 
-const isAdviceMeeting = (enquiryMethod) => enquiryMethod === 'Meeting';
+const isAdviceMeeting = (enquiryMethod) => enquiryMethod.toLowerCase() === 'meeting';
```

Comment:
```suggestion
const isAdviceMeeting = (enquiryMethod) => enquiryMethod?.toLowerCase() === 'meeting';
```

I just checked and prod does have a few nulls for `s51Advice.method` so this would throw an error. We can add a null check and all good!

Reply:
good spot, will do!